### DISCUSSION TITLE
GIF Animation for TensorBoard on Segmentation Tasks

### DISCUSSION THREAD

**Discussion by jpcenteno80** [Project-MONAI/MONAI#2505] [Ideas]
Hi, for segmentation tasks, I think the GIF animation for TensorBoard with "3D" tensors would be more useful if the segmentation mask could be placed as a transparency over the input tensor, and instead of grayscale for the mask, use colors... Many times the segmentation mask GIF is out of sync with the input image GIF which makes it hard to visually assess the quality of the model.
![image](https://user-images.githubusercontent.com/22669584/124173615-83f11180-da68-11eb-82b1-60d4439b580e.png)

**Comment by wyli** [comment]
thanks for raising this issue, it is possible to add some postprocessing transforms to achieve it. please see this (slightly out of date) notebook https://github.com/Project-MONAI/MONAIBootcamp2020/blob/master/day1notebooks/lab3_post_transforms.ipynb
